### ISSUE TITLE
Support Moonriver XCM

### ISSUE THREAD

**Issue by bvotteler** [interlay/bridge#40]
Support XCM between Moonriver and Kintsugi for:
- [ ] MOVR
- [ ] kBTC
- [ ] KINT
- [ ] stKSM

Notes:
- stKSM is blocked on Lido at the moment.
- Extract stKSM to separate (future work) issue if still blocked by the time work for this issue has commenced.

**Comment by tomjeatt** [comment]
Blocked by https://github.com/interlay/interbtc-ui/issues/1376 which will allow us to support Moonbeam address formats.
